clients/turbo.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": [".next/*", "!.next/cache/*"]
+    },
+    "dev": {
+      "dependsOn": ["build"],
+      "outputs": [".next/*", "!.next/cache/*"]
+    },
+    "start": {
+      "dependsOn": ["build"],
+      "outputs": [".next/*", "!.next/cache/*"]
+    },
+    "lint": {
+      "dependsOn": []
+    },
+    "lint:fix": {
+      "dependsOn": []
+    },
+    "format": {
+      "dependsOn": []
+    },
+    "format:ci": {
+      "dependsOn": []
+    },
+    "test": {
+      "dependsOn": ["build"]
+    },
+    "test:ci": {
+      "dependsOn": ["build"]
+    },
+    "cy:open": {
+      "dependsOn": ["build"]
+    },
+    "cy:run": {
+      "dependsOn": ["build"]
+    },
+    "cy:start": {
+      "dependsOn": ["build"]
+    },
+    "analyze": {
+      "dependsOn": ["build"]
+    },
+    "analyze:browser": {
+      "dependsOn": ["build"]
+    },
+    "analyze:server": {
+      "dependsOn": ["build"]
+    },
```

Comment:
these might not either, but I'm less sure about that

Reply:
you're right, I've removed!